pkg/apis/apps/types.go

```diff
@@ -97,6 +97,33 @@ type RollingUpdateStatefulSetStrategy struct {
 	Partition int32
 }
 
+// PersistentVolumeClaimDeletePolicyType is a string enumeration of the policies for
+// deletion of VolumeClaimTemplates when the controlling StatefulSet is deleted or
+// scaled down.
+type PersistentVolumeClaimDeletePolicyType string
+
+const (
+	// RetainPersistentVolumeClaimDeletePolicyType is the default
+	// PersistentVolumeClaimDeletePolicy and specifies that PersistentVolumeClaims
+	// associated with StatefulSet VolumeClaimTemplates will not be deleted when the
+	// controlling StatefulSet is deleted or scaled down.
+	RetainPersistentVolumeClaimDeletePolicyType PersistentVolumeClaimDeletePolicyType = "Retain"
```

Comment:
Unfortunately, comments for the enum values don't make it to the reference docs. So you need to include the list of values, a small description and the default in the field comment.

Reply:
This had been done back in March.